wpilibc/src/main/native/include/frc/I2C.h

```diff
@@ -135,7 +135,7 @@ class I2C : public ErrorBase {
   bool VerifySensor(int registerAddress, int count, const uint8_t* expected);
 
  private:
-  HAL_I2CPort m_port;
+  HAL_I2CPort m_port = HAL_I2C_kOnboard;
```

Comment:
HAL_I2CPort is an enum rather than a handle, so I thought HAL_kInvalidHandle wasn't a valid option. Now that I'm looking at this HAL_I2C_kOnboard is 0, which is the same value as HAL_kInvalidHandle. HAL_CloseI2C() will produce a no-op if the port is greater than 1, but that's only because the comment in there `// Set port out of range error here` hasn't been implemented. Wouldn't HAL_kInvalidHandle being -1 make more sense than 0?

Reply:
Let's keep track of it in the I2C class for now and open an issue to fix the underlying HAL concern.

Reply:
Done.